src/Faker/Provider/tk_TM/Person.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Faker\Provider\tk_TM;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @var array Turkmen person name formats.
+     */
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}a',
+    );
+
+    /**
+     * @var array Turkmen first names.
+     */
+    protected static $firstNameMale = array(
```

Comment:
This list, like the two following ones, is a little bit too small for Faker. Please find a larger dataset to insure a good variety in generated names.

Also, please mention the datasource in a phpDoc comment to ease future updates.